### ISSUE TITLE
Feature request: Price comparison

### ISSUE THREAD

**Issue by zunami** [grocy/grocy#1653]
A price comparison of were **before** my shopping would be great.

Example:
You open the brochure and find cans of corn at
Dealer A: € 1,69 
Dealer B: € 1,29
Dealer C: € 0,79 Action ...

you can then create a shopping list based on the data or also look up for later queries how the prices move.

Input options that I think of
* Time limited action prices
* incl. quantities advantages
* Shelf prices

**Comment by newtogithub-mike** [comment]
good idea

**Comment by DevEDVColin** [comment]
Great Idea👍👍

**Comment by initd0** [comment]
I was about to open an issue/feature request for this as i am looking for the same thing. Also, ive been looking for a self-hosted solution just for this tbh, and stumbled upon Grocy. Although all the current features are amazing and perfect, this feature is definitely a must and probably (depending on the logic) deserves its own category on the left hand pane, such as "Shopping Pricing Preparation" (unless you can integrate it somewhere else)

Say a user wants to "Prepare for researching prices on a Specific Product"

Building on what @zunami said,

- When the product is being added, the user must be able to add as many "Dealers/Shops" as he wants/needs, (including the Dealer's details, address, notes, etc.)
- Eventually when the product is added, i want to be able to see each Dealer's name and price together in one page, and compare to see which place im going to buy said product from with specified date of price added!
- Historical views of how prices have changed after long period would be nice to see
- Would be nice to point out in a colour like "green" which Dealer is the cheapest

There is something else that would be great but not sure if it can be considered part of this feature.
"Blacklisting of Dealers/Shops"
Basically, would be good to know you researched many dealers/shops, but blacklisted a few on a specified date because of any reasons which would be placed under the dealers/shops notes.

Would appreciate this price comparison feature very much! and thank you for your hard work

**Comment by rosscullen** [comment]
In the USA, there's an app called Basket https://basket.com
It's trying to do the above but reviews are mixed. Worth looking at the screenshots to give you a visual on what its trying to achieve

**Comment by initd0** [comment]
> In the USA, there's an app called Basket https://basket.com It's trying to do the above but reviews are mixed. Worth looking at the screenshots to give you a visual on what its trying to achieve

Thats exactly right.. This feature would be a real plus if it can be implemented properly within grocy. Not to mention having a historic view of what you paid for and compared back in the day.  Manually finding prices from different stores from thorough research for a specific item is tiring as it is, keeping them all documented to make a final decision is whats needed for sure.

**Comment by protonaut** [comment]
I order bigger amounts of dryed goods (grain, nuts, etc.) online. Because of the delivery costs I don't want to order each product separetly. 
There are plenty of sellers on ebay for example. The prices differ trough different products. 
How can I optimize the buying process of getting the right seller for the needed goods or split into multiple orders if that's cheaper.

Is there already a software out for this request or is  there a plan of integrating into the grocy-app?

**Comment by berrnd** [comment]
@protonaut please use reactions to express that you're also interested, comments adding nothing useful for the topic in question creates nothing but useless noise, thanks.

&nbsp;
> Is there already a software out for this request

Pretty the wrong place for that here

&nbsp;
> is there a plan of integrating into the grocy-app

There are no plans for anything, this is a hobby project ([README ref](https://github.com/grocy/grocy#roadmap)).

**Comment by protonaut** [comment]
Thanks for the reply. Is there any discussion board or wiki for sharing user experience? Maybe that's supposed to be a better the place for my request.

**Comment by berrnd** [comment]
> Is there any discussion board or wiki for sharing user experience?

Spamming again the totally wrong place, cool. Simply have a look at README: https://github.com/grocy/grocy#questions--help--bug-reports--feature-requests

Or the website: https://grocy.info/#getting-in-touch

And since I know reading is more than hard, here the direct link to the r/grocy subreddit:
# &rarr; https://www.reddit.com/r/grocy
